### ISSUE TITLE
Some questions about the Visualization Section in the paper.

### ISSUE THREAD

**Issue by STEVEN-PP** [zezhishao/STID#6]
Dear shao, in your paper, 
> For _T DiW_ ∈ R𝑁𝑤 ×𝐷 , where 𝑁𝑤 = 7 ≪ 𝐷 = 32, we train STID by setting the embedding size of _T DiW_ to 2 to get a more accurate visualization.

And in your code, ` # concate all embeddings
hidden = torch.cat([time_series_emb] + node_emb + tem_emb, dim=1) `

My confusion is how to concatenate the three embeddings in the given dimension when training the model if you change the embedding size of _T DiW_ to 2  whileas the size of other embeddings is still 32.
Other embeddings size change to 2 also? Or something else?

**Comment by zezhishao** [comment]
The dimensions of `time_series_emb`, `node_emb` and `tem_emb` do not need to be the same.

**Comment by zezhishao** [comment]
In the visualization section, we train a new STID by setting the dimension of day_in_week embedding to 2 and leaving the other dimensions unchanged.

**Comment by STEVEN-PP** [comment]
Yes, u r right! Thanks!
